resources/js/admin/auth/passwords/change.js

```diff
@@ -0,0 +1,6 @@
+import Vue from "vue";
+import AvatarImage from "../../../components/AvatarImage.vue";
+import VuePassword from "vue-password";
+
+Vue.component("avatar-image", AvatarImage);
+Vue.component("vue-password", VuePassword);
```

Comment:
Add new line at the end of the file.
```suggestion
Vue.component("vue-password", VuePassword);

```